docs/02-app/01-building-your-application/07-configuring/03-environment-variables.mdx

```diff
@@ -161,7 +161,7 @@ export default function Component() {
 - You can run code on server startup using the [`register` function](/docs/app/building-your-application/optimizing/instrumentation).
 - We do not recommend using the [runtimeConfig](/docs/pages/api-reference/next-config-js/runtime-configuration) option, as this does not work with the standalone output mode. Instead, we recommend [incrementally adopting](/docs/app/building-your-application/upgrading/app-router-migration) the App Router.
 
-## Default Environment Variables
+## Default Environments
```

Comment:
This still feels funny to me. What about any of these?

* Local Environment Variables
* Configuring local Environment Variables
* Managing local Environment Variables